storage/imx6ull-flash/flashdrv.c

```diff
@@ -782,6 +800,11 @@ int flashdrv_read(flashdrv_dma_t *dma, uint32_t paddr, void *data, flashdrv_meta
 {
 	int chip = 0, channel = 0, sz = 0, result;
 	char addr[5] = { 0 };
+
+	if (flashdrv_addrPrep(&paddr, &chip) < 0) {
+		return -1;
```

Comment:
this actually returns raw uint32_t HW register (AFAIK no caller explicitly checks it's bit fields, but surprisingly it might have first bit set and be negative?)

Reply:
So it's OK to set all bits in retval then